weapon-config.inc

```diff
@@ -824,7 +824,7 @@ static s_LastVehicleEnterTime[MAX_PLAYERS];
 static s_TrueDeath[MAX_PLAYERS];
 static s_InClassSelection[MAX_PLAYERS];
 static s_ForceClassSelection[MAX_PLAYERS];
-static s_ClassSpawnInfo[300][E_SPAWN_INFO];
```

Comment:
I always thought max classes were the same as total skins....i guess not. I set it to 319 now.